payments-core/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     implementation project(":hcaptcha")
     implementation project(":stripe-ui-core")
     compileOnly project(':financial-connections')
+    implementation project(':financial-connections-lite')
```

Comment:
The proposal was to add this to `paymentsheet`. But I see this adds it to `payments-core`. What's the impact here, and why is this necessary?